Sources/TuistGraph/DependenciesGraph/DependenciesGraph.swift

```diff
@@ -6,7 +6,7 @@ import TuistSupport
 
 public enum DependenciesGraphError: FatalError, Equatable {
     /// Thrown when the same dependency is defined more than once.
-    case duplicatedDependency(String, ExternalDependency, ExternalDependency)
+    case duplicatedDependency(String, [TargetDependency], [TargetDependency])
```

Comment:
why arrays? 🤔 do you want to list all duplicated dependencies? if so Id rename this error:

`duplicatedDependency` -> `duplicatedDependencies`

documentation update is needed as well

Reply:
The type of `externalDependencies` is `[String: [TargetDependency]]`, so this must be the value of that dictionary.
The `externalDependencies` need an array as value, because one SPM product might contain multiple targets, which would be mapped to multiple `TargetDependency.project(target:path)` dependencies